charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -87,5 +87,23 @@ spec:
       volumeSnapshotLocation: []
     metrics:
       serviceMonitor:
+        enabled: true
         additionalLabels: {{- .Values.monitoring.labels | toYaml | nindent 10 }}
+      prometheusRule:
+        enabled: true
+        spec:
+          - alert: VeleroBackupPartialFailures
+            annotations:
+              message: Velero backup {{ "{{ $labels.schedule }}" }} has {{ "{{ $value | humanizePercentage }}" }} partialy failed backups.
+            expr: velero_backup_partial_failure_total{schedule!=""} / velero_backup_attempt_total{schedule!=""} > 0.25
+            for: 15m
+            labels:
+              severity: warning
+          - alert: VeleroBackupFailures
+            annotations:
+              message: Velero backup {{ "{{ $labels.schedule }}" }} has {{ "{{ $value | humanizePercentage }} "}} failed backups.
+            expr: velero_backup_failure_total{schedule!=""} / velero_backup_attempt_total{schedule!=""} > 0.25
+            for: 15m
+            labels:
+              severity: warning
```

Comment:
We should overthink this, these alarms are fading over time

We should use the metric `velero_backup_last_status`, if it's working now (in the old version the status wasn't always correct... f*ck velero 🤮)

Reply:
I just copied it from the chart values.yaml :sweat_smile: 
So I guess it does not work as expected :rofl:

Reply:
Do we even want to deploy rules?

Everyone could need a different set of rules, so i'm not sure if we want this.

Reply:
> Do we even want to deploy rules?

Definitely, otherwise monitoring this is useless

> Everyone could need a different set of rules, so i'm not sure if we want this.

These rules will be generic if we use `velero_backup_last_status`, therefore they work for everyone

The only thing we can't monitor is, that backups are attempted when they should, but that's a currently unsolvable problem, like with "alert if a desired cluster is missing"..., as we don't have metrics for "should exist"

Reply:
I have implemented the requested changes